sdk/src/server-api/sc/framework/plugins/AbstractGame.kt

```diff
@@ -16,32 +13,37 @@ import sc.shared.ScoreCause
 import sc.shared.WinCondition
 import java.util.*
 
-abstract class RoundBasedGameInstance<P : Player>(@XStreamOmitField override val pluginUUID: String) : IGameInstance {
+abstract class AbstractGame<P : Player>(override val pluginUUID: String) : IGameInstance {
     companion object {
-        val logger = LoggerFactory.getLogger(RoundBasedGameInstance::class.java)
+        val logger = LoggerFactory.getLogger(AbstractGame::class.java)
     }
-
+    
+    override val players = mutableListOf<P>()
+    
     var activePlayer: P? = null
         protected set
-
-    @XStreamAsAttribute
-    var turn = 0
-        private set
-    open val round: Int
-        get() = turn / 2
-
-    @XStreamOmitField
-    private var paused: Int? = null
-
-    @XStreamOmitField
-    private var moveRequestTimeout: ActionTimeout? = null
-
-    @XStreamOmitField
+    
     protected val listeners = mutableListOf<IGameListener>()
-
-    @XStreamImplicit(itemFieldName = "player")
-    override val players = mutableListOf<P>()
-
+    
+    private var moveRequestTimeout: ActionTimeout? = null
+    
+    var paused = false
+        private set
+    
+    val isPaused: Boolean
+        get() = paused
```

Comment:
I'm not satisfied with `paused`, `isPaused` and `setPauseMode`.
It feels like they could all be merged together, no?

Reply:
I thought about it, yes, I wan't sure myself